package.json

```diff
@@ -17,21 +17,27 @@
     "@next-auth/prisma-adapter": "^1.0.3",
     "@prisma/client": "3.14.0",
     "axios": "^0.27.2",
+    "bootstrap": "^5.1.3",
```

Comment:
Do we need `bootstrap`?

Reply:
Doesn't look like it
- no imports for it
- everything ran fine without it

Bootstrap removed in 407069a